test/JUnit.Xml.TestLogger.AcceptanceTests/JUnitTestLoggerAcceptanceTests.cs

```diff
@@ -96,6 +95,32 @@ public void TestResultFileShouldContainTestCases()
             Assert.AreEqual(6, skips.Count());
         }
 
+        [TestMethod]
+        public void TestCasesShouldContainStandardOut()
+        {
+            var node = this.resultsXml.XPathSelectElements("/testsuites/testsuite").Descendants();
+            var testcases = node.Where(x => x.Name.LocalName == "testcase").ToList();
+
+            var testCasesWithSystemOut = testcases
+                .Where(x => x.Descendants().Any(y => y.Name.LocalName == "system-out"))
+                .ToList();
+
+            Assert.AreEqual(2, testCasesWithSystemOut.Count());
+        }
+
+        [TestMethod]
+        public void TestCasesShouldContainErrorOut()
+        {
+            var node = this.resultsXml.XPathSelectElements("/testsuites/testsuite").Descendants();
+            var testcases = node.Where(x => x.Name.LocalName == "testcase").ToList();
+
+            var testCasesWithErrorOut = testcases
+                .Where(x => x.Descendants().Any(y => y.Name.LocalName == "system-err"))
+                .ToList();
+
+            Assert.AreEqual(1, testCasesWithErrorOut.Count());
```

Comment:
@Siphonophora I'm having trouble getting system-err to work on a per-testcase basis. Here I would expect the stderr from `FailTest11()` to be found. Any ideas why it's not getting picked up?

Reply:
@mclayton7 I suspect its due to how nunit, which we use for these tests, is behaving. Looking at their docs, they make the distinction between standard output whose messages go to the test level and errors which get displayed right away. https://docs.nunit.org/articles/nunit/writing-tests/TestContext.html 

Based on what we see in the loggers, I think the immediate messages are coming across as test messages https://github.com/spekt/testlogger/blob/master/src/TestLogger/Core/TestRunMessageWorkflow.cs but are not attached to the results. While normal console.writeline is going to both the standard out and is being attached to the test result. 

I can look more later in the week, but if i'm right then this behavior will depend on the test library and we won't be able to do much on the logging side.

Reply:
@Siphonophora I think you're correct. I can remove the stderr logging here since it doesn't work, and just leave the stdout.

Reply:
I think it would be good if we could do a quick test with mstest and xunit, so we can put a note in the docs about how each behaves.